src/MarginTrading.DataReader/Startup.cs

```diff
@@ -105,6 +110,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             app.UseAuthentication();
             app.UseMvc();
             app.UseSwagger();
+            app.UseSwagger(DocumentFilter, "swagger/{apiVersion}/swagger-no-api-key.json");
```

Comment:
2 times UseSwagger?

Reply:
Yes. This provides two swagger urls with different content. Swagger-no-api-key.json is needed for the autorest generator: we add an api-key header as a parameter to all methods in swagger, but we don't need it to be generated by autorest. In DocumentFilter we strip it as it was not present, and generate autorest client using the swagger-no-api-key.json.